app/src/main/java/org/p2p/wallet/swap/ui/orca/OrcaSwapPresenter.kt

```diff
@@ -677,13 +680,13 @@ class OrcaSwapPresenter(
             fromToken = sourceTokenSymbol,
             toToken = destinationTokenSymbol
         )
-        transactionManager.emitTransactionState(state)
+        transactionManager.emitTransactionState(transactionId, state)
```

Comment:
i think there is no need for a lateinit field
you create transactionId only in this method.

pass transactionId in showError as a parameter, because showError can be called without swap called before